packages/framework/src/Form/Admin/Holidays/HolidaysImportFormType.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Form\Admin\Holidays;
+
+use Override;
+use Shopsys\FormTypesBundle\ActionBarType;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Form\DomainsType;
+use Shopsys\FrameworkBundle\Model\Holiday\HolidaysImportData;
+use Shopsys\FrameworkBundle\Model\Holiday\HolidaysImportFacade;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Validator\Constraints\Callback;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+
+final class HolidaysImportFormType extends AbstractType
+{
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Holiday\HolidaysImportFacade $holidaysImportFacade
+     * @param \Symfony\Component\Routing\RouterInterface $router
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
+     */
+    public function __construct(
+        private readonly HolidaysImportFacade $holidaysImportFacade,
+        private readonly RouterInterface $router,
+        private readonly Domain $domain,
+    ) {
+    }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     * @param array $options
+     */
+    #[Override]
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('countryCode', ChoiceType::class, [
+                'label' => 'Country',
+                'choices' => $this->holidaysImportFacade->getValidCountryChoices(),
+                'placeholder' => 'Select country',
+                'required' => true,
+                'help_html' => true,
+                'help' => t('If the country is not listed, you have to <a href="%countryCreateUrl%" target="_blank">create</a> it first. Or, in rare scenarios, your country code might not be <a href="%supportedCountriesUrl%" target="_blank">supported</a>.', [
+                    '%countryCreateUrl%' => $this->router->generate('admin_country_new'),
```

Comment:
just a question - If I don't have a permission, the link will be properly generated, I just get 403 error, right?
I can live with that